bolt.gemspec

```diff
@@ -48,7 +48,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "net-ssh", ">= 4.0"
   spec.add_dependency "net-ssh-krb", "~> 0.5"
   spec.add_dependency "orchestrator_client", "~> 0.4"
-  spec.add_dependency "puppet", [">= 6.16.0", "< 6.18.0"]
+  spec.add_dependency "puppet", [">= 6.16.0", "< 6.19.0"]
```

Comment:
We should probably set the minimum version to 6.18.0, since Bolt will no longer work with < 6.18.0.